runtime/v2/runner/src/main/java/com/walmartlabs/concord/runtime/v2/runner/script/DefaultScriptEvaluator.java

```diff
@@ -37,153 +37,184 @@
 import java.io.BufferedWriter;
 import java.io.Reader;
 import java.io.Writer;
-import java.util.Collections;
-import java.util.LinkedList;
-import java.util.List;
-import java.util.Map;
```

Comment:
intellij did this, not i!

Reply:
Check if you're using the default code style:
![image](https://user-images.githubusercontent.com/826959/219893152-056ad265-79af-401e-a3c8-0a85cba9bfec.png)

Some plugins might also interfere here.